src/util/triplea/MapXMLCreator/MapXMLCreator.java

```diff
@@ -0,0 +1,839 @@
+		        break;
+					case 15:
+						final String[] settings = GameSettingsPanel.s_allSettings.split(", ");
+						StringBuilder sb = new StringBuilder();
+						sb.append("<html><table border=0><tr>");
+						int setting_nr = 1;
+						for (final String setting : settings)
+						{
+							sb.append("<td>" + setting + "</td>");
+							if (setting_nr % 3 == 0)
+								sb.append("</tr><tr>");
+							++setting_nr;
+						}
+						sb.append("</tr></table></html>");
+						showInfoMessage(
+									"Setting Name: The name of the setting that is applied to the map. Examples: Always on AA, Two hit battleship, and Japanese bid.\r\n\r\nValue: The value of the game setting. Examples: true, false, 0, 5, 32.\r\n\r\nEditable: Whether players are allowed to change the value of the setting when the map is being started.\r\n\r\nMin. N. (Optional): The lowest number that the value can be set to when the user sets the game options in TripleA. Only change this if the setting is a number.\r\n\r\nMax. N.(Optional): The highest number that the value can be set to when the user sets the game options in TripleA. Only change this if the setting is a number.\r\n\r\nTo have the program automatically enter some of the commonly used Game Settings, click the 'Auto-Fill' button between the Back and Next buttons.",
+									"Help For Current Step");
+		        break;
+		        case 16: showInfoMessage("The map notes can be entered in the textbox. You can enter plain text or html code. Then just click the 'Save Map To File' button, and save the file in the map's 'games' folder. (Create the 'games' folder if not already created)", "Help For Current Step");
+		        break;
+				}
+			}
+		});
+		southLeftPanel.add(m_bHelp);
+		
+		southCenterPanel = new JPanel();
+		m_southPanel.add(southCenterPanel);
+		
+		m_southPanel.add(Box.createHorizontalGlue());
+		
+		m_bBack = new JButton("Back");
+		m_bBack.setMnemonic(KeyEvent.VK_B);
+		m_bBack.setFont(new Font("Tahoma", Font.PLAIN, 11));
+		m_bBack.addActionListener(new AbstractAction("Previous Step")
+		{		
+			private static final long serialVersionUID = -8623081809358073307L;
+
+			public void actionPerformed(final ActionEvent e)
+			{
+				SwingUtilities.invokeLater(new Runnable()
+				{
+					public void run()
+					{
+						if (m_currentStep > 1)
+							goToStep(m_currentStep-1);
+						m_actionPanel.validate();
+						m_actionPanel.repaint();
+					}
+				});
+			}
+		});
+		southCenterPanel.add(m_bBack);
+		
+		m_bAuto = new JButton("Auto-Fill");

```

Comment:
Is it possible to have this variable be a methold local, ie: `JButton autoFillButton = ...`
This would avoid having to track a lot of state around. Complex class state makes for complex classes. Minimizing scope of all variables helps.